tools/deployment/package/otel-collector-config.yaml

```
@@ -0,0 +1,35 @@
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+    endpoint: "0.0.0.0:13133"
```

YAMLlint (1.38.0)

[error] duplication of key "endpoint" in mapping

(key-duplicates)